src/app/app-routing.module.ts

```diff
@@ -2,11 +2,16 @@ import { NgModule } from '@angular/core';
 import { RouterModule, Routes } from '@angular/router';
 import { CartComponent } from './component/cart/cart.component';
 import { ProductsComponent } from './component/products/products.component';
+import { LoginComponent } from './login/login.component';
+import { SignupComponent } from './signup/signup.component';
 
 const routes: Routes = [
   {path:'', redirectTo:'products',pathMatch:'full'},
   {path:'products', component: ProductsComponent},
-  {path:'cart', component: CartComponent}
+  {path:'cart', component: CartComponent},
+  { path: 'login', component: LoginComponent },
+  { path: 'signup', component: SignupComponent },
+  { path: '', redirectTo: '/login', pathMatch: 'full' }, 
```

Comment:
So this will add new flow to the app that it will always redirect to the login page, which is fine to me.
But can we remove the above `{path:'', redirectTo:'products',pathMatch:'full'},`